test/verify/machineslib.py

```diff
@@ -1810,7 +1816,10 @@ def installWithErrorTest(dialog):
             self.machine.execute(" && ".join(cmds))
 
             # Add an extra network interface that should appear in the PXE source dropdown
-            iface = self.add_iface(activate=False)
+            self.machine.upload(["verify/files/veth.sh"], "/tmp/")
+            veth = self.machine.spawn("/tmp/veth.sh", "veth")
+            self.addCleanup(self.machine.execute, "kill {0} || true".format(veth))
+            iface = "eth42"
```

Comment:
This is fine for initial testing of course, but for the final version this needs to move to testlib.py, similar to `add_disk()`, and not hardcode the name. It also doesn't need to be a spawned process, as it does nothing. It should just addCleanup the teardown of the interface name.